pkg/destroy/libvirt/libvirt.go

```diff
@@ -68,6 +68,15 @@ func (o *ClusterUninstaller) Run() error {
 	return nil
 }
 
+// New returns libvirt Uninstaller from ClusterMetadata.
```

Comment:
All other New funcs are the first function after the uninstraller func in this PR, maybe follow that for this too?

Reply:
resolved